rb/lib/selenium/webdriver/common/script.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+module Selenium
+  module WebDriver
+    class Script
```

Comment:
Don't we want to namespace this under `BiDi` module?

Reply:
As "High Level API" no
But, I'm starting to think we don't want High Level API to call directly to websocket, and should go through a BiDi namespaced class. Let me resubmit an idea

Reply:
The thing with `Script` class specifically is that it is going to hold implementations for both the `script` and the `log` domains (based on our conversation at the summit). So we do want to call down to something lower level rather than make it be the lower level class.